.github/workflows/ci.yml

```diff
@@ -70,8 +72,11 @@ jobs:
               with:
                   php-version: ${{ matrix.php-version }}
 
+            - name: Install Symfony Flex
+              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex
```

Comment:
why do you need flex?

Reply:
To enforce Symfony version

Reply:
ok, I missed the line where you export `SYMFONY_REQUIRE` ; sorry !